tests/network/sriov.go

```diff
@@ -520,6 +523,46 @@ var _ = Describe("[Serial]SRIOV", func() {
 			})
 		})
 
+		Context("Connected to multiple SRIOV networks", func() {
+			BeforeEach(func() {
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet1, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet2, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet3, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet4, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+			})
+
+			It("should create a virtual machine with sriov interfaces with custom MAC addresses referring the appropriate resource", func() {
```

Comment:
How about: "should create a virtual machine with **SR-IOV** interfaces with custom MAC addresses referring **to the correct PCI address**"?

Reply:
Done